### DISCUSSION TITLE
v3.11.0

### DISCUSSION THREAD

**Discussion by rabilrbl** [rabilrbl/jiotv_go#443] [Releases]
## What's Changed
* chore(deps): Bump github.com/schollz/progressbar/v3 from 3.15.0 to 3.16.0 by @dependabot in https://github.com/JioTV-Go/jiotv_go/pull/440
* [HOTFIX]: Remove gzip from API requests by @itsyourap in https://github.com/JioTV-Go/jiotv_go/pull/441

**Full Changelog**: https://github.com/JioTV-Go/jiotv_go/compare/v3.10...v3.11.0

<hr /><em>This discussion was created from the release <a href='https://github.com/JioTV-Go/jiotv_go/releases/tag/v3.11.0'>v3.11.0</a>.</em>

**Comment by varkey** [comment]
Looks like the container images are no longer available for the newer versions. The latest version tagged image that I see is 3.9.5.

https://github.com/users/rabilrbl/packages/container/package/jiotv_go

@rabilrbl

**Reply by itsyourap** [reply]
All docker users must change image to `ghcr.io/jiotv-go/jiotv_go`
Old docker image will no longer be updated

**Reply by varkey** [reply]
Ah I see, I didn't notice that change. Thanks for the quick response.

**Comment by ngr-ch** [comment]
PS C:\JioTV> .\jiotv_go-windows-386_5.exe run
INFO: No config file found, using environment variables

 ┌───────────────────────────────────────────────────┐
 │                     JioTV Go                      │
 │                   Fiber v2.52.5                   │
 │               http://127.0.0.1:5001               │
 │                                                   │
 │ Handlers ............ 48  Processes ........... 1 │
 │ Prefork ....... Disabled  PID ............. 10196 │
 └───────────────────────────────────────────────────┘

PS C:\JioTV>

**Reply by ngr-ch** [reply]
Not working any version

**Reply by itsyourap** [reply]
Use `debug = true` in your jiotv_go.toml config file, then run jiotv_go and give us the log.

**Reply by ngr-ch** [reply]
Working fine after fresh installation..Thanks Bro..
